new-starter-guide/2016-03-15-new-starter-to-origami.md

```diff
@@ -0,0 +1,134 @@
+# New starter guide
+
+Welcome to Origami!
+
+## Checklist of things to do
+
+The checklist assumes that you have done the following:
+
+1. Visit IT Service Desk to get your computer account up and running.
+2. Logged on your own work machine and email account.
+
+### Admin privileges on your machine
+
+IT Services should have enabled admin privileges when they have set up your machine but you should [request admin access to your FT laptop via Salesforce](https://financialtimes.my.salesforce.com/home/home.jsp) to ensure it does not get revoked.
+
+If you do not have admin privileges then email `itservicedesk@ft.com`. Include your desk number (which is usually on the left of the desk) and the machine number with the email. You also will need to say why you need admin privileges, or alternatively copy the template below and edit accordingly.
+
+```
+Dear IT Service Desk,
+
+I am a new starter within the Origami team, as a <your job title>. I would like to have admin privileges on <machine number> which is at <desk number>.
+
+The reasons for needing access to admin privileges is so that I can install software such as node.js, <your preferred text editor>, <add more if you like> and etc.
+
+I look forward to hearing back from you on this matter.
+
+Regards
+
+<your name>
+```
+
+This should expedite your request for admin privileges on your machine.
+
+
+### Slack
+
+1. Install Slack (This can be done without admin privileges).
+  > To install certain apps without admin privileges, you can access the Self Service app facility by finding it on your machine via Spotlight (under the assumption it's a Mac) Log in with your network details, and do a search for the Slack app and install. This is the same with browsers like Google Chrome or Mozilla Firefox.
+
+2. The Slack domain for FT is `financialtimes`. Log into the domain with your FT email address.
+
+3. Join the `#ft-origami` channel and ask for invites to the internal channels from the Origami team.
+
+
+### Code repositories
+
+1. The public repositories are found at [GitHub/Financial-Times](https://github.com/Financial-Times) on GitHub. The private ones are found at [BitBucket](http://git.svc.ft.com/). The Origami repositories should be public if at all possible.
```

Comment:
we also have private repos on github and we're looking into moving them all to github